genienlp/run_bootleg.py

```diff
@@ -96,10 +96,10 @@ def parse_argv(parser):
     )
 
     parser.add_argument(
-        '--min_entity_len', type=int, default=1, help='Minimum length for entities when ngrams database_lookup_method is used '
+        '--min_entity_len', type=int, default=1, help='Minimum token-length of entities in ngram-based lookup of bootleg'
```

Comment:
"ngram-based lookup" is implementation detail, the point is that entities shorter than that will not be retrieved